arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -71,7 +71,8 @@ class ffmpeg_video_input::priv
     number_of_frames(0),
     collected_all_metadata(false),
     estimated_num_frames(false),
-    sync_metadata(true)
+    sync_metadata(true),
+    max_seek_back_attempts(10)
```

Comment:
Not important, but have you considered changing these over to inline member initializers?

Reply:
I assume you would like all of the variables moved to inline initialization? What is the reasoning behind this for my edification?

Reply:
It's probably better to do all or none, yes.

Inline initializers propagate to all constructors that don't override them (less duplication). You don't have to type the member name twice. Having the declaration and initial value together may be clearer. It removes the possibility of messing up the initialization order. It is more diff-friendly because you don't need to add a comma to the preceding initializer. It may help ensure that members are initialized, since one can check for this at the point of declaration rather than having to compare (potentially multiple) constructor(s) to declarations.

The [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-in-class-member-initializers-instead) further claim they may be more efficient, although I'm unsure of the logic behind that.

Reply:
Ugh - just realized you probably wanted them initialized at declaration.

Reply:
Ok - now I think I got it.